### ISSUE TITLE
Difference between develop branch and PyPi

### ISSUE THREAD

**Issue by mabergerx** [pyannote/pyannote-audio#1424]
I am trying to use `pyannote.audio` in an air-gapped environment with its own PyPi proxy, such that I can perform a `pip install pyannote.audio`.

However, using this method I encounter all kinds of problems that I can't reproduce on a Google Colab using `pip install -qq https://github.com/pyannote/pyannote-audio/archive/refs/heads/develop.zip`. In this scenario, both running the voice activity pipeline using HuggingFace loader and using the local model work fine. 

So these are the scenarios that work:

**Google Colab**
1) Install through `pip install -qq https://github.com/pyannote/pyannote-audio/archive/refs/heads/develop.zip`.
2) Then, instantiate the pipeline and run it
```python
from pyannote.audio import Pipeline
from pyannote.audio import Inference
import numpy as np

from IPython.display import Audio

audio_filepath = "/content/my_file.mp3"

# Login with HF credentials, etc, and then...
pipeline = Pipeline.from_pretrained("pyannote/voice-activity-detection")

initial_params = {"onset": 0.9, "offset": 0.8,
                  "min_duration_on": 0.05537587440407595, "min_duration_off": 0.09791355693027545}
pipeline.instantiate(initial_params)

# This works
audio_timeline = pipeline(audio_filepath).get_timeline()
```

...and also this scenario:

```python
# Upload the model file and config.yaml to Colab, and then...

from pyannote.audio import Pipeline
from pyannote.audio import Inference
import numpy as np

from IPython.display import Audio

audio_filepath = "/content/my_file.mp3"

offline_vad = Pipeline.from_pretrained("/content/config.yaml")

initial_params = {"onset": 0.9, "offset": 0.8,
                  "min_duration_on": 0.05537587440407595, "min_duration_off": 0.09791355693027545}
offline_vad.instantiate(initial_params)

# This also works
audio_timeline = offline_vad(audio_filepath)
```

Now here are the scenarios that don't work:

**My air-gapped environment**
1) Install through `pip install pyannote.audio`
2) Upload the segmentation model files used in the example above to the environment and trying to load them:
```python
from pyannote.audio import Pipeline
from pyannote.audio import Inference

import numpy as np

from IPython.display import Audio

audio_filepath = "my_file.mp3"

offline_vad = Pipeline.from_pretrained("config.yaml")

initial_params = {"onset": 0.9, "offset": 0.8,
                  "min_duration_on": 0.05537587440407595, "min_duration_off": 0.09791355693027545}
offline_vad.instantiate(initial_params)

# Loading the pipeline works, it loads without errors and returns an instance of a VAD pipeline, but then...

audio_timeline = offline_vad("my_file.mp3")
>>> RuntimeError: Error opening 'my_file.mp3': Error : bad map offset.
```

**Google Colab**
1) Install through `pip install pyannote.audio`
2) Upload the segmentation model files used in the example above to the Colab and trying to load them:
```python
# This instantly gives an error
from pyannote.audio import Pipeline
>>> OSError: /usr/local/lib/python3.10/dist-packages/torchtext/lib/libtorchtext.so: undefined symbol: _ZN2at4_ops10select_int4callERKNS_6TensorElN3c106SymIntE
```

**In summary**, when installing the library from the develop branch on Google Colab, I can use both HF initialization and local files to produce the desired results. However, when using the PyPi version, I get errors on both my own environment and on Colab, even though the errors are different, while the data stays the same.  Seeing as I can't easily install the develop branch on my environment from GitHub, I wonder why the behaviour I see occurs.

Thanks.

**Comment by triinity2221** [comment]
Hi @mabergerx, I have the same issue were you able to resolve it
